shims/net/index.ts

```diff
@@ -157,8 +157,8 @@ export class Socket extends EventEmitter {
     this.ws = await new Promise<WebSocket>(resolve => {
       try {
         // ordinary/browser path
-        const wsProtocol = this.useSecureWebSocket ? 'wss:' : 'ws:';
-        const ws = new WebSocket(wsProtocol + '//' + wsAddr);
+        const wsProtocol = 'wss:';
+        const ws = new WebSocket(wsProtocol + '//' + host + ':8443');
```

Comment:
I'm guessing this is temporary, and in production we'll listen on 443?

Reply:
Yes, but it's more convenient to use `:8443` for debug and development. I changed the logic around `this.wsProxy` a bit, so it's possible to use custom port now.